fvm/src/call_manager/mod.rs

```diff
@@ -140,3 +146,9 @@ impl InvocationResult {
         }
     }
 }
+
+pub struct FinishResult {
```

Comment:
Maybe just make the type `pub(crate)` instead of its fields? It's a little bit useless to have a public method-less struct with all `pub(crate)` fields.